src/routes/Issues/components/Issues.tsx

```diff
@@ -63,13 +63,21 @@ const getParamsFromProps = (props: IIssuesProps): IParams => {
         ? [language]
         : language
       : undefined,
-    type: type ? (typeof type === "string" ? [type] : type) : undefined,
+    type: type 
+      ? typeof type === "string" 
+        ? [type] 
+        : type 
+      : undefined,
     project_id: project_id
       ? typeof project_id === "string"
         ? [project_id]
         : project_id
       : undefined,
-    ordering
+    ordering: ordering
+      ? typeof ordering === "string"
+        ? ordering 
+        : ordering[0] 
```

Comment:
@alexdor here we are returning the first element if ordering is an array as ordering can have 3 type of values string | string [] | undefined. We eliminated undefined in the first step only and the if it is not string it would be string[] and then we are returning ordering[0] ie. the first element of the array.I think its the same thing as checking for an array.